lib/rubocop/comment_config.rb

```diff
@@ -32,9 +32,7 @@ def extra_enabled_comments
     end
 
     def comment_only_line?(line_number)
-      non_comment_token_line_numbers.none? do |non_comment_line_number|
-        non_comment_line_number == line_number
-      end
+      non_comment_token_line_numbers.none?(line_number)
```

Comment:
Do we have a cop for this and `any?` already?

Reply:
Yes. RuboCop Performance has `Performance/RedundantEqualityComparisonBlock` cop introduced by https://github.com/rubocop/rubocop-performance/pull/213.

https://docs.rubocop.org/rubocop-performance/1.10/cops_performance.html#performanceredundantequalitycomparisonblock